demos/unity_states/cs/generated/cpp_sharp/Std.cs

```diff
@@ -0,0 +1,14 @@
+// ----------------------------------------------------------------------------
```

Comment:
Let's `.gitignore` the `generated` directory.

Reply:
done